2do_fe/src/components/AddTask/AddTaskPopUp.js

```diff
@@ -43,57 +43,78 @@ function AddTaskLogo() {
 }
 
 function TaskDescForm() {
+  const [taskData, setTaskData] = useState({
+    status: "in progress",
+    title: "",
+    description: "",
+    dueDate: null,
+  });
+
+  const dispatch = useDispatch();
+
+  const handleChange = (e) => {
+    setTaskData({ ...taskData, [e.target.name]: e.target.value});
+  }
+
   return (
     <center>
       <div id="Form-container">
         <form>
           <label htmlFor="title">Title</label>
-          {/* <input
-        type="text"
-        className="input-title"
-        placeholder="Enter task title"
-        // name = "title"
-        // id = "title" />
-      /> */}
           <TextField
-            type="text"
+            required
             className="textfieldstyle"
-            id="title"
+            type="text"
+            name="title"
+            vlaue={taskData.title}
```

Comment:
Think this should be value?

Reply:
You're right. That was my bad. Which parts need more clarification?

Reply:
Just fixed the typo!